cmd/backup/storages/local.go

```diff
@@ -0,0 +1,118 @@
+package storages
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+	"time"
+
+	t "github.com/offen/docker-volume-backup/cmd/backup/types"
+	u "github.com/offen/docker-volume-backup/cmd/backup/utilities"
+	"github.com/sirupsen/logrus"
+)
+
+type LocalStorage struct {
```

Comment:
Why is this "meta-type" that just embeds another type needed?

Reply:
Do you mean in general or in the specific case of the `LocalStorage`?